libethereum/Account.cpp

```diff
@@ -155,7 +156,7 @@ AccountMap dev::eth::jsonToAccountMap(std::string const& _json, u256 const& _def
                         cerr << "Error importing code of account " << a
                              << "! Code needs to be hex bytecode prefixed by \"0x\".";
                     else
-                        ret[a].setCode(fromHex(codeStr));
+                        ret[a].setCode(fromHex(codeStr), 0);
```

Comment:
I'll create an issue to handle this properly. We should have the ability to set contract's version in config file.

Reply:
https://github.com/ethereum/aleth/issues/5645